sentence_transformers/fit_mixin.py

```diff
@@ -14,8 +14,8 @@
 from torch.utils.data import DataLoader
 from tqdm.autonotebook import trange
 from transformers import TrainerCallback, TrainerControl, TrainerState
+from transformers.utils import is_datasets_available
```

Comment:
(no strong opinion)
I would likely reimplement this here rather than import a utility from an external library. Given you're not using the version, you can have a simple implementation

```py
import importlib.util

spec = importlib.util.find_spec("datasets")
return spec is not None

Reply:
Sounds good. I was already thinking about combining `is_datasets_available` and `is_accelerate_available` into an `is_training_available` as I have to use both for some checks in several locations, so it makes sense to implement it all locally.